examples/gltf/main.rs

```diff
@@ -206,9 +206,6 @@ fn toggle_or_cycle_animation(
                 );
             }
             scene.animation_index += 1;
-            if scene.animation_index == 6 {
-                scene.animation_index = 7;
-            }
```

Comment:
Why did you remove that one?

Reply:
I guess animation 6 was skipped because it was broken as single frame animation, so we didnt skip it anymore because its fixed.

Reply:
:+1: